index.html

```diff
@@ -8,7 +8,7 @@
       href="https://fonts.googleapis.com/css?family=Poppins"
       rel="stylesheet"
     />
-    <link rel="icon" type="image/*" href="assets/images/portfolio.png" />
+    <link rel="icon" type="image/*" href="assets/images/portfolio.png" alt="Portfolio"/>
     <link rel="stylesheet" href="./assets/style/main.css" />
     <title>Desktop version Portfolio</title>
```

Comment:
I suggest that you use your name or profession as a title as this is your protfolio desktop version is not referring to the project correctly